src/wallet/crypter.cpp

```diff
@@ -151,6 +151,23 @@ static bool DecryptSpendingKey(const CKeyingMaterial& vMasterKey,
     return sk.address() == address;
 }
 
+static bool DecryptSaplingSpendingKey(const CKeyingMaterial& vMasterKey,
+                               const std::vector<unsigned char>& vchCryptedSecret,
+                               const libzcash::SaplingFullViewingKey& fvk,
+                               libzcash::SaplingSpendingKey& sk)
+{
+    CKeyingMaterial vchSecret;
+    if(!DecryptSecret(vMasterKey, vchCryptedSecret, fvk.GetHash(), vchSecret))
+        return false;
+
+    if (vchSecret.size() != libzcash::SerializedSpendingKeySize)
```

Comment:
This is technically incorrect, because it's the Sprout spending key size. But in practice both Sprout and Sapling spending keys serialize to the same size. If another reviewer wants it changed, I'll support that.

Reply:
Will add `const size_t SerializedSaplingSpendingKeySize = 32;` and use that, for clarity